startup/gui/menus.py

```diff
@@ -381,6 +381,7 @@ def __shaderNodeCreator( nodeName, shaderName ) :
 nodeMenu.append( "/Image/Utility/Stats", GafferImage.ImageStats, searchText = "ImageStats", postCreator = GafferImageUI.ImageStatsUI.postCreate  )
 nodeMenu.append( "/Image/Utility/Sampler", GafferImage.ImageSampler, searchText = "ImageSampler" )
 nodeMenu.append( "/Image/Utility/Catalogue", GafferImage.Catalogue )
+nodeMenu.append( "/Image/Utility/CatalogueSelect", GafferImage.CatalogueSelect )
```

Comment:
Elsewhere we're representing the camel case names as space-separated labels, like so :

```nodeMenu.append( "/Image/Utility/Catalogue Select", GafferImage.CatalogueSelect, searchText = "CatalogueSelect" )```